packages/dashboard-core-plugins/tsconfig.json

```diff
@@ -34,6 +34,9 @@
     {
       "path": "../iris-grid"
     },
+    {
+      "path": "../jsapi-bootstrap"
+    },
     {
       "path": "../jsapi-shim"
     },
```

Comment:
We should really have some sort of eslint/rule to test that these match the package.json... need to add jsapi-types here.
Can also remove all these extra new lines, they're not necessary and makes the file more readable without them imo.